CHANGELOG_iOS.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+## [1.63.183](https://github.com/brave/brave-browser/releases/tag/v1.63.183)
+
+### Web3  
+- Implemented v2 UI for "Account" details in Brave Wallet. ([#8663](https://github.com/brave/brave-ios/issues/8663))
+- Implemented v2 UI for “Accounts” tab in Brave Wallet. ([#8600](https://github.com/brave/brave-ios/issues/8600))
+- Fixed transaction status not updating automatically in "Transaction Details" page. ([#36468](https://github.com/brave/brave-browser/issues/36468))
+
+### General
+
+- Added Brave Leo - AI for questions and answers leveraging page context. ([#36585](https://github.com/brave/brave-browser/issues/36585))
+- Added “Copy Clean Link” to the share menu. ([#8070](https://github.com/brave/brave-ios/issues/8070))
+- Added interstitial pages when blocking pages (top-level domain blocking). ([#8096](https://github.com/brave/brave-ios/issues/8096))
+- Added “Open in Brave” to iOS Share Sheet menu. ([#8516](https://github.com/brave/brave-ios/issues/8516))
+- Added opening scanned URLs in a new tab when selecting “Scan QR Code” from the Home Screen context menu. ([#8594](https://github.com/brave/brave-ios/issues/8594))
+- Removed the limit on the number of filter lists that can be selected at once in “Content Filtering” page. ([#8656](https://github.com/brave/brave-ios/issues/8656))
```

Comment:
hmm I think `under` mind sound better in this case. For example, something similar to:

```
- Removed the limit on the number of filter lists that can be selected at once under “Content Filtering” settings page. ([#8656](https://github.com/brave/brave-ios/issues/8656))
```